djconnectwise/models.py

```diff
@@ -1071,6 +1071,13 @@ class Meta:
     def __str__(self):
         return self.description
 
+    @property
+    def api_class(self):
+        return api.ProjectAPIClient
+
+    def _update_cw(self, api_client, updated_objects):
```

Comment:
Updating through the model is the VERY old pattern, we don't do that at all anymore. The only ones that are left with that is just for legacy reasons. Everything should be kept in the synchronizer and API classes